README.md

```diff
@@ -23,26 +23,55 @@ in an experimental and research phase and cannot currently be used traditionally
 source code base for general changes.
 ```
 
-**This repo does not include any assets or code necessary for compiling the ROM. A prior copy of the game is required to extract the required assets.**
+This is a WIP **decompilation** of ***The Legend of Zelda: Majora's Mask***. The purpose of the project is to recreate a source code base for the game from scratch, using information found inside the game along with static and/or dynamic analysis. **It is not producing a PC port.** For more information you can get in touch with the team on our [Discord server](https://discord.zelda64.dev).
 
-This is a decompilation of Legend of Zelda: Majora's Mask (US) 1.0
+The only build currently supported is N64 US, but other versions are planned to be supported.
 
-It builds the following ROM:
+It currently builds the following ROM:
 * mm.us.rev1.rom.z64 `md5: 2a0a8acb61538235bc1094d297fb6556`
 
+**This repo does not include any assets or assembly code necessary for compiling the ROM. A prior copy of the game is required to extract the required assets.**
+
 Please refer to the following for more information:
 
 - [Website](https://zelda64.dev/)
 - [Discord](https://discord.zelda64.dev/)
 - [How to Contribute](CONTRIBUTING.md)
 
+
 ## Installation
 
 ### Windows
 
 For Windows 10, install WSL and a distribution by following this
 [Windows Subsystem for Linux Installation Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
-We recommend using Debian or Ubuntu 18.04 Linux distributions.
+We recommend using Debian or Ubuntu 20.04 Linux distributions.
+
+
+### MacOS
+
+For macOS, use Homebrew to install the following dependencies:
+
+* coreutils
+* make
+* python3
+* libpng
+
+You can install them with the following commands:
+
+```bash
+brew update
+brew install coreutils make python3 libpng
+```
+
+(The repository expects Homebrew-installed programs to be either linked correctly in `$PATH` etc. or in their default locations.)
+
+You'll also need to [build and install mips-linux-binutils](docs/BUILDING_BINUTILS_MACOS.md).
+
+The makefile should alias Apple's out-of-date `make` to `gmake` automatically, but if you have issues, try using `gmake` in place of `make`.
```

Comment:
From what I understand, the default `make` command in Mac machines isn't out of date, it is just another variation/flavour of Makefile. For this project we use GNU Makefile specifically, that's why Mac users require to use `gmake` instead.

Reply:
No, the reason it's different is because Apple refuse to use anything GPLv3-licensed, so they have by default the last GPLv2 version of make:
```
$ make -v
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
$ gmake -v
GNU Make 4.3
Built for arm-apple-darwin21.1.0
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```